### ISSUE TITLE
Publish Docker Containers

### ISSUE THREAD

**Issue by ohdearaugustin** [juanfont/headscale#88]
As mentioned as a TODO in the README of the k8s folder. 
We should build containers like we make releases with github actions.

Therefore I was so free and registered the headscale user on hub.docker.com. (Which I would hand over)
Just wanted to make sure we will have that one. (https://hub.docker.com/u/headscale)

The idea is to publish the headscale docker containers on

-  ghrc.io
-  hub.docker.com

In short there will be a PR, which implements the needed workflows. I can afterwards help to setup the whole thing.
